deep-map/index.js

```diff
@@ -0,0 +1,15 @@
+import { atom } from '../atom/index.js'
+import { getPath, setPath } from './path.js'
+
+export * from './path.js'
```

Comment:
Can we move this function here? Otherwise this file become too small 😅

Reply:
Currently modules are separated by their function. I surely can merge them, but it'll both make type notations harder to read/edit and the module itself will be messier.

> file become too small

… so?